tests/queries/0_stateless/03324_parallel_parsing_zero_bytes_buffer.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+CUR_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+#  shellcheck source=../shell_config.sh
+. "$CUR_DIR"/../shell_config.sh
+
+$CLICKHOUSE_LOCAL -q "desc file('$CUR_DIR/data_bson/c0_Int_10.bson')"
+
+$CLICKHOUSE_LOCAL -q """
+    DROP TABLE IF EXISTS t0;
+    CREATE TABLE t0 (c0 Int) ENGINE = MergeTree() PRIMARY KEY (c0);
+    INSERT INTO TABLE t0 (c0) SELECT * FROM file('$CUR_DIR/data_bson/c0_Int_10.bson');
```

Comment:
I initially wrote this test by just replicating the test case and didn't want to remove this afterwards.